pom.xml

```diff
@@ -7,10 +7,48 @@
     <groupId>com.srutkowski.training</groupId>
     <artifactId>unittest</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <name>unittest</name>
 
     <properties>
         <maven.compiler.target>11</maven.compiler.target>
         <maven.compiler.source>11</maven.compiler.source>
     </properties>
 
+    <dependencies>
+       <dependency>
+        <groupId>org.junit.jupiter</groupId>
+       <artifactId>junit-jupiter-engine</artifactId>
+       <version>5.8.2</version>
+       <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.mockito</groupId>
+       <artifactId>mockito-core</artifactId>
+       <version>4.2.0</version>
+       <scope>test</scope>
+      </dependency>
+
+		<dependency>
+			<groupId>org.hamcrest</groupId>
+			<artifactId>hamcrest-library</artifactId>
+			<version>2.2</version>
+			<scope>test</scope>
+		</dependency>
+
+      <dependency>
+         <groupId>org.assertj</groupId>
+         <artifactId>assertj-core</artifactId>
+         <version>3.22.0</version>
+      </dependency>
+
+      <dependency>
+    <groupId>org.mockito</groupId>
+    <artifactId>mockito-junit-jupiter</artifactId>
+    <version>4.2.0</version>
+    <scope>test</scope>
+</dependency>
```

Comment:
Minor: Pay attention to the proper indents for the nested elements in all source code files. The code will be more readable. For example, line 46 should start with the bigger indent than 45th, and all <dependency> tags should be on the same level